src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -422,6 +425,44 @@ def setPassword(self, newPassword):
         else:
             raise Exception('Unknown error '+ hex(receivedPacketPayload[0]))
 
+    def handShake(self):
```

Comment:
I would like to name the method `testHandShake()`. What do you mean @philippmeisberger ?

Reply:
I would prefer `handshake()`. `testHandShake()` sounds like unit-test ;)

Reply:
Okay, i agree

Reply:
@jacozsz Please name the method `handshake()` (all lower case) ;)